docs/cs/beginners/index.md

```diff
@@ -0,0 +1,66 @@
+    content: Začátečníci | Ethereum
+---
+
+# Vítejte! {#-welcome}
+
+Slyšíte o Ethereu poprvé? Jste na správném místě. Pojďme na to.
+
+**Ethereum je základem nové éry internetu:**
+
+- Internet, do něhož jsou zabudovány peníze a platby.
+- Internet, kde jsou uživatelé pány svých dat a aplikace nešpehují a nekradou.
+- Internet, kde má každý přístup k otevřenému finančnímu systému.
+- Internet postavený na neutrální, otevřené a nikým nekontrolované infrastruktuře.
+
+**Ethereum, založené v roce 2015, je předním programovatelným blockchainem.**
+
+**Stejně jako ostatní blockchainy, i Ethereum má svou vlastní měnu - Ether (ETH). **Ether jsou digitální peníze. Možná jsi slyšel o [Bitcoinu](http://bitcoin.org/) - ETH má mnoho podobných vlastností. Je čistě digitální, a lze ji okamžitě poslat komukoli na světě. Nabídku ETH neříká žádná vláda ani společnost - je decentralizovaná a omezená. Lidé na celém světě využívají ETH k platbám, k uložení hodnoty nebo jako zálohu.
+
+**Ale narozdíl od ostatních blockchainů, Ethereum toho umí mnohem víc.** Ethereum je programovatelné, což znamená že ho mohou využít vývojáři pro [nové druhy aplikací](/cs/use/#1-use-an-application-built-on-ethereum).
+
+Tyto decentralizované aplikace (nebo také "dapps") mají výhody kryptoměny a blockchainové technologie. Mohou být důvěryhodné, což znamená že jakmile jsou "uploadovány" na Ethereum, budou vždy fungovat tak, jak byly naprogramovány. Mohou kontrolovat digitální aktiva a tím vytvářet nové druhy aplikací. Mohou být decentralizované, což znamená že je žádna entita nebo osoba nemůže kontrolovat.
+
+**Právě teď, tisíce developerů po celém světe staví a vytváří aplikace na Ethereu, které můžete využít právě dnes:**
+
+- [**Peněženky pro kryptoměny**](/cs/use/#_3-what-is-a-wallet-and-which-one-should-i-use), které umožňují levné, rychlé platby s ETH nebo jinými aktivy
+- **Finanční aplikace**, které umožňují půjčit nebo investovat vaše digitální aktiva
+- **Decentralizované trhy**, které umožňují měnit digitální aktiva, nebo dokonce "předpovědi" událostí v reálném světě
+- **Hry**, kde lze vlastnit in-game aktiva a dokonce i vydělat reálné peníze
+- **A mnohem**, mnohem víc.
+
+**The Ethereum community is the largest and most active blockchain community in the world.** It includes core protocol developers, cryptoeconomic researchers, cypherpunks, mining organizations, ETH holders, app developers, ordinary users, anarchists, fortune 500 companies, and, as of now, **you**.
+
+**There is no company or centralized organization that controls Ethereum.** Ethereum is maintained and improved over time by a diverse global community of contributors who work on everything from the core protocol to consumer applications. This website, just like the rest of Ethereum, was built - and continues to be built - by a collection of people working together.
+
+**Welcome to Ethereum.**
+
+**Not sure where to go next?**
+
+- Want to get started using Ethereum? [ethereum.org/use](/cs/use/)
```

Comment:
We'll want to update this link text as well (i.e. ethereum.org/cs/use vs. ethereum.org/use)